src/scripts/librarymenu.js

```diff
@@ -12,6 +12,7 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
         html += "</div>";
         html += '<div class="headerRight">';
         html += '<span class="headerSelectedPlayer"></span>';
+        html += '<button is="paper-icon-button-light" class="headerSyncButton syncButton headerButton headerButtonRight hide"><i class="material-icons">sync_disabled</i></button>';
```

Comment:
```suggestion
        html += '<button is="paper-icon-button-light" class="headerSyncButton syncButton headerButton headerButtonRight hide"><span class="material-icons sync_disabled"></span></button>';
```
we're going to replace `<i>` for `<span>` for icons, let's use `<span>` here from the get go. cc @dmitrylyzo